docs/cloud/insights/events-feed.md

```diff
@@ -0,0 +1,59 @@
+| **Auditing events** - COMING SOON<br/>Events related to actions done on your Space, e.g. invite user, change user role or change plan.| 4 hours | 7 days | 90 days |
+| **[Topology events](#topology-events)**<br/>Node state transition events, e.g. live or offline.| 4 hours | 7 days | 14 days |
+| **[Alert events](#alert-events)**<br/>Alert state transition events, can be seen as an alert history log.| 4 hours | 7 days | 90 days |
+
+##### Topology events
+
+| **Event name** | **Description** |  **Example** |
+| :-- | :-- | :-- |
+| Node Become Live | The node is collecting and streaming metrics to Cloud.| Node `netdata-k8s-state-xyz` is on live **state** |
+| Node Become Stale | The node is offline and not streaming metrics to Cloud. It can show historical data from a parent node. | Node `ip-xyz.ec2.internal` is on **stale** state |
+| Node Become Offline | The node is offline, not streaming metrics to Cloud and not available in any parent node.| Node `ip-xyz.ec2.internal` is on **offline** state |
+| Node Created | The node is created but it is still Unseen on Cloud, didn't establish a successful connection yet.| Node `ip-xyz.ec2.internal` is on **unseen** state |
+| Node Removed | The node was removed from the Space through the `Delete` action, if it becomes Live again it will be automatically added. | Node `ip-xyz.ec2.internal` was **deleted(soft)** |
+| Node Restored | The node is restored, if node becomes Live after a remove action it is re-added to the Space. | Node `ip-xyz.ec2.internal` was **restored** |
+| Node Deleted | The node is deleted from the Space, see this as a hard delete and won't be re-added to the Space if it becomes live. | Node `ip-xyz.ec2.internal` was **deleted(hard)** |
+| Agent Claimed | The agent was successfully registered to Netdata Cloud and is able to connect.<br/>These events can only be seen on _All nodes_ War Room. | Agent with claim ID `7d87bqs9-cv42-4823-8sd4-3614548850c7` was claimed. |
+| Agent Connected | The agent connected to the Netdata Cloud MQTT server (Agent-Cloud Link established).<br/>These events can only be seen on _All nodes_ War Room. | Agent with claim ID `7d87bqs9-cv42-4823-8sd4-3614548850c7` has connected to Cloud. |
+| Agent Disconnected | The agent disconnected from the Netdata Cloud MQTT server (Agent-Cloud Link severed).<br/>These events can only be seen on _All nodes_ War Room. | Agent with claim ID `7d87bqs9-cv42-4823-8sd4-3614548850c7` has disconnected from Cloud: **Connection Timeout**. |
+| Agent Authenticated | The agent successfully authenticated itself to Netdata Cloud.<br/>These events can only be seen on _All nodes_ War Room. | Agent with claim ID `7d87bqs9-cv42-4823-8sd4-3614548850c7` has successfully authenticated. |
+| Agent Authentication Failed | The agent failed to authenticate itself to Netdata Cloud.<br/>These events can only be seen on _All nodes_ War Room. | Agent with claim ID `7d87bqs9-cv42-4823-8sd4-3614548850c7` has failed to authenticate. |
```

Comment:
These two events are not currently emitted.

Reply:
@ralphm which two? the highlight is showing four. was also checking [the ticket](https://github.com/netdata/cloud-frontend/issues/4015) were the mappings were added and we have 3 as not yet emitted
* agent-authenticated
* agent-authentication-failed
* agent-claimed

the one not emitted are just the authentication related ones?

Reply:
There's no highlight, there's just 4 lines between this and the previous comment 😄

The authentication are not yet being emitted and neither is agent claimed.

Reply:
removed